pkg/asset/rhcos/bootstrap_image.go

```diff
@@ -0,0 +1,54 @@
+// Package rhcos contains assets for RHCOS.
+package rhcos
+
+import (
+	"context"
+	"time"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/rhcos"
+	"github.com/openshift/installer/pkg/types/baremetal"
+)
+
+// BootstrapImage is location of RHCOS image.
+// This stores the location of the image based on the platform.
+// eg. on AWS this contains ami-id, on Livirt this can be the URI for QEMU image etc.
+type BootstrapImage string
```

Comment:
This needs to be appropiately named to map this is the boot image for bootstrap-host. the comments should mention that.

Reply:
Done